### ISSUE TITLE
Warnings in listen 3.0.2 + jruby

### ISSUE THREAD

**Issue by deivid-rodriguez** [guard/listen#330]
I'm getting the following warnings. Just posting them here so you can evaluate if there's something to consider or it's better to just live with them

```
/home/davidr/.gem/jruby/2.2.2/gems/listen-3.0.2/lib/listen/event/queue.rb:15: warning: toplevel constant Queue referenced by Thread::Queue

/home/davidr/.gem/jruby/2.2.2/gems/listen-3.0.2/lib/listen/event/loop.rb:16: warning: toplevel constant Queue referenced by Thread::Queue
```

Regards!

**Comment by e2** [comment]
Thanks - this was reported before (in another issue), but I couldn't reproduce it.

I still couldn't reproduce it, but it should be fixed.

Can you confirm this branch fixes it?

```ruby
gem 'listen', github: 'guard/listen', branch: 'e2-fix_toplevel_const_warning'
```

**Comment by deivid-rodriguez** [comment]
It does! Thanks!!

**Comment by lustremedia** [comment]
Same warnings here. Any stable release where that is fixed?

Using listen (3.0.2)
Using guard (2.12.8)

**Comment by e2** [comment]
@lustremedia - released as 3.0.3 enjoy!

**Comment by lustremedia** [comment]
@e2 perfect, thanks!
